devnet/.envrc

```diff
@@ -15,6 +15,11 @@ export COMMITTEE_CANDIDATE_ADDRESS=$(jq -r '.addresses.CommitteeCandidateValidat
 export D_PARAMETER_POLICY_ID=$(jq -r '.mintingPolicies.DParameterPolicy' devnet/addresses.json)
 export PERMISSIONED_CANDIDATES_POLICY_ID=$(jq -r '.mintingPolicies.PermissionedCandidatesPolicy' devnet/addresses.json)
 
+# native token observability
+export PARTNER_CHAIN_NATIVE_TOKEN_POLICY_ID='ada83ddd029614381f00e28de0922ab0dec6983ea9dd29ae20eef9b4'
+export PARTNER_CHAIN_NATIVE_TOKEN_ASSET_NAME='5043546f6b656e44656d6f'
+export PARTNER_CHAIN_ILLIQUID_SUPPLY_VALIDATOR_ADDRESS='addr_test1wrhvtvx3f0g9wv9rx8kfqc60jva3e07nqujk2cspekv4mqs9rjdvz'
```

Comment:
This will be inconsistent. We don't use PARTNER_CHAIN_ prefix for our configs. There is exception to BLOCK_BENEFICIARY, but as you can see it didn't aged well.

Reply:
You're right, I removed it